package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "marlowe-runner",
+  "version": "0.1.0",
```

Comment:
This must be added otherwise the nix build will complain about the `marlowe-runner` package missing a version field (and it's good practice anyway)